src/components/availability-panel/drag-drop-context/index.tsx

```diff
@@ -87,8 +87,13 @@ const DragDropcontext: FC<dragDropProps> = ({
         const assignee = result.combine.droppableId === 'tasks'
           ? result.draggableId
           : result.combine.draggableId;
+        const dateObject:any = new Date();
```

Comment:
Let's try to drop the `any` wherever possible. Also, every class or interface can be used as a type in TypeScript.

So this line can be written as,
`const dateObject = new Date();`

Reply:
Got it will update it